build.cake

```diff
@@ -128,7 +128,7 @@ Task("DotnetPublish")
     var portablePublishDir =  $"{octoPublishFolder}/portable";
     DotNetCorePublish(projectToPublish, new DotNetCorePublishSettings
     {
-        Framework = "netcoreapp3.1",
+        Framework = "netcoreapp2.0",
```

Comment:
Can we add a comment as to why this says `netcoreapp2.0` (otherwise people will go "that's obviously wrong, and break things")

Reply:
Good idea. Done.